activerecord/lib/active_record/connection_adapters/mysql/schema_definitions.rb

```diff
@@ -7,6 +7,10 @@ def primary_key(name, type = :primary_key, **options)
           super
         end
 
+        def blob(*args, **options)
```

Comment:
`t.binary "tiny_blob", limit: 255` is generated as `tiny_blob varchar(255)`. We need the type for correctly dumping the `tinyblob`.